frontend/src/scenes/events/eventsTableLogic.ts

```diff
@@ -104,10 +154,18 @@ export const eventsTableLogic = kea({
             },
         ],
         events: [
-            [],
+            [] as EventType[],
             {
-                fetchEventsSuccess: (state, { events, isNext }) => (isNext ? [...state, ...events] : events),
-                prependNewEvents: (state, { events }) => [...events, ...state],
+                fetchEventsSuccess: (state: EventType[], { events, isNext }: OnFetchEventsSuccess) =>
+                    isNext ? [...state, ...events] : events,
+                prependNewEvents: (
+                    state: EventType[],
+                    {
+                        events,
+                    }: {
+                        events: EventType[]
+                    }
+                ) => [...events, ...state],
```

Comment:
and the same here and elsewhere, no need to do any `{ events: EventType[] }`. All of this should come automatically.